packages/big-design/src/components/Dropdown/Dropdown.tsx

```diff
@@ -4,26 +4,42 @@ import { Manager, Reference, RefHandler } from 'react-popper';
 import scrollIntoView from 'scroll-into-view-if-needed';
 
 import { uniqueId } from '../../utils';
+import { Flex } from '../Flex';
+import { FlexItem } from '../Flex/Item';
+import { Link } from '../Link';
 import { List } from '../List';
-import { ListItem } from '../List/Item';
+import { ListItem, ListItemProps } from '../List/Item';
 
 interface DropdownState {
   highlightedItem: HTMLLIElement | null;
   isOpen: boolean;
 }
 
-interface Props {
+export interface DropdownProps extends Omit<React.HTMLAttributes<HTMLUListElement>, 'children'> {
   maxHeight?: number;
+  options: Array<DropdownItem | DropdownLinkItem>;
   placement?: Placement;
   trigger: React.ReactElement;
-  onItemClick?(value: string | number | Array<string | number>): void;
 }
 
-export type DropdownProps = Props & React.HTMLAttributes<HTMLUListElement>;
+export interface DropdownItem extends BaseItem {
+  type?: 'string';
+}
 
-export class Dropdown extends React.PureComponent<DropdownProps, DropdownState> {
-  static Item = ListItem;
+export interface DropdownLinkItem extends BaseItem {
+  target?: HTMLAnchorElement['target'];
+  type: 'link';
+  url: string;
+}
 
+interface BaseItem extends Omit<ListItemProps, 'children' | 'content' | 'onClick' | 'value'> {
```

Comment:
Can you move the `BaseItem` definition before `DropdowItem` and `DropdownLinkItem` :pray: